files/usr/share/cinnamon/cinnamon-menu-editor/cme/util.py

```diff
@@ -78,46 +78,46 @@ def getUniqueUndoFile(filepath):
             break
     return new_filepath
 
-def getItemPath(file_id):
+def getItemPath(file_id) -> str | None:
```

Comment:
This syntax is only supported from 3.10+, you need to use `Optional` to support 3.6+

Reply:
@Gr3q , @mtwebster thanks for the feedback. 
I changed the `str | None` to `Optional[str]` in all places. Now it should support 3.6+